packages/trader/src/Modules/Contract/Containers/contract-replay.jsx

```diff
@@ -151,6 +151,7 @@ ContractReplay.propTypes = {
     history: PropTypes.object,
     indicative_status: PropTypes.string,
     is_chart_loading: PropTypes.bool,
+    is_chart_fitting: PropTypes.bool,
```

Comment:
can we call it `is_chart_scaling` instead?
```suggestion
    is_chart_scaling: PropTypes.bool,
```

Reply:
@zaki-binary handled